places/models.py

```diff
@@ -1,14 +1,38 @@
 from django.db import models
 
+from easy_maps.geocode import google_v3 as geocode
+
 
 class Place(models.Model):
     name = models.CharField(max_length=100, unique=True)
     description = models.CharField(max_length=450)
-    address_street = models.CharField(max_length=120)
+    address_country = models.CharField(max_length=60)
     address_city = models.CharField(max_length=60)
+    address_street = models.CharField(max_length=120)
     address_post_code = models.CharField(max_length=8)
     date_created = models.DateTimeField(auto_now=False, auto_now_add=True)
     date_updated = models.DateTimeField(auto_now=True, auto_now_add=False)
+    latitude = models.DecimalField(max_digits=16, blank=True,
+                                   decimal_places=8)
+    longitude = models.DecimalField(max_digits=16, blank=True,
+                                    decimal_places=8)
 
     def __unicode__(self):
         return self.name
+
+    def get_full_address(self):
+        """
+        create string with full address
+        """
+        return unicode(self.address_country) + \
```

Comment:
Ten import akurat niewiele pomoże, unicode() dalej będzie próbował zdekodować stringa za pomocą ascii, więc jeżeli trafi się string zakodowany w utf8 to się wysypie.

No tylko pytanie jakie są szanse, że faktycznie trafi tam string a nie unicode, bo od django zawsze dostajemy unicode.